packages/core/scripts/gas-rebate/index.js

```diff
@@ -219,20 +276,10 @@ async function parseClaimEvents(claimedRewards, priceData, multibar, rebateOutpu
   };
 }
 
-async function calculateRebate(_startDate, _endDate, _revealOnly, _claimOnly) {
+async function calculateRebate({ rebateNumber, startBlock, endBlock, revealOnly, claimOnly }) {
   try {
     const voting = new web3.eth.Contract(VotingAbi.abi, "0x9921810C710E7c3f7A7C6831e30929f19537a545");
```

Comment:
I know this is from before but should we not pull this from the artifacts?

Reply:
Yeah we should